### PR TITLE
Get Supported Tickers v2 Deprecation

### PR DESCRIPTION
**Pull request by clarkpd**

### CODE REVIEW

**Comment by clarkpd** [comment]
fixes issue #22

File: src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/Tickers.kt [hunk 1]
```diff
@@ -58,57 +78,87 @@ data class SupportedTickersParameters(
...
     val active: Boolean = false,
-    @SerialName("primaryExch") val primaryExchange: String? = null,
-    @SerialName("updated") val lastUpdated: String? = null,
-    val url: String? = null,
-    val codes: Map<String, String> = emptyMap(),
-    val attrs: Map<String, String> = emptyMap()
+    val cik: String? = null,
+    @SerialName("composite_figi") val compositeFigi: String? = null,
+    @SerialName("currency_name") val currencyName: String? = null,
+    @SerialName("last_updated_utc") val lastUpdatedUtc: String? = null,
```
Comment (mmoghaddam385) [inline src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/Tickers.kt:156]: Looks like we're missing the `delisted_utc` field in the DTO

```suggestion
    @SerialName("last_updated_utc") val lastUpdatedUtc: String? = null,
    @SerialName("delisted_utc") val delistedUtc: String? = null,
```

**Review by mmoghaddam385 — COMMENTED** [review]
Thanks so much for contributing, this PR looks great! I only noticed that the updated TickerDTO is missing the `delisted_utc` field from the documentation so I made a comment suggestion for that.

**Review by mmoghaddam385 — APPROVED** [review]
Thanks again for contributing! Since this is a breaking change we'll release this as a major version update. I will close your issue once the release is out!

**Pull request merged**